actiontext/lib/generators/action_text/install/install_generator.rb

```diff
@@ -9,7 +9,7 @@ class InstallGenerator < ::Rails::Generators::Base
       source_root File.expand_path("templates", __dir__)
 
       def install_javascript_dependencies
-        rails_command "app:update:bin"
+        rails_command "app:update:bin", inline: true
```

Comment:
This line was not hit by the tests I ran locally, so I am currently unsure of its impact.